Lib/test/test_zipfile/test_core.py

```diff
@@ -47,6 +47,28 @@ def get_files(test):
         yield f
         test.assertFalse(f.closed)
 
+
+def create_zipfile_with_extra_data(filename, extra_data_name):
```

Comment:
Would you mind moving it to the instance method of AbstractTestsWithSourceFile?

Reply:
class `OtherTests` isn't inherited from `AbstractTestsWithSourceFile`. Should I move helper  and tests to `AbstractTestsWithSourceFile`?

Reply:
Done.